src/generic_ui/scripts/ui.ts

```diff
@@ -134,6 +134,25 @@ export interface NotificationData {
   unique ?:string;
 }
 
+var PROXYING_SESSION_ID_LENGTH = 16;
+
```

Comment:
Only because I think I'll need it a few lines later if the promise rejects:
https://github.com/uProxy/uproxy/blob/trevj-proxying-id/src/generic_ui/scripts/ui.ts#L609

Alternatively, we could define a new `uproxy_core_api.Update` and listen for that, as we do for `FRIEND_FAILED_TO_GET`. What do you think? We could call it `FAILED_TO_GET` (I think `FRIEND_FAILED_TO_GET` should be renamed `FAILED_TO_GIVE`)